test/e2e/must-gather/must_gather_suite_test.go

```diff
@@ -62,12 +62,13 @@ var _ = ginkgo.BeforeSuite(func() {
 	mustGatherTag = getStringValueFromEnv(envVarMustGatherTag, defaultMustGatherTag)
 	ginkgo.By(fmt.Sprintf("Using must-gather image %q tag %q", mustGatherImage, mustGatherTag))
 
+	ginkgo.By("Fetching up cluster data")
```

Comment:
this seems wrong

Reply:
indeed. I'll make it "collect available data" does that sound better?